Polindrom/script.js

```diff
@@ -0,0 +1,19 @@
+function palindrome(str) {
+
```

Comment:
![image](https://user-images.githubusercontent.com/29356908/144327108-4eeb1941-1869-42ec-b21c-f4aed44fc831.png)

Reply:
in my opinion that is bug. 
 here is more difficult the expression:
**_palindrome("My age is 0, 0 si ega ym.") should return true._**
and everything works